### DISCUSSION TITLE
Rust Graphics Meetup #2

### DISCUSSION THREAD

**Discussion by AngelOnFira** [rust-gamedev/wg#123] [Ideas]
I'm going to try and run a second Rust Graphics Meetup. I should be able to head it up on my own, but I need some speakers :) The general objective is to get people who work on different parts of the Rust Graphics ecosystem to give some technical talks. Or maybe there are other topics that would also do nicely, we'll see who applies :+1: 

I'm going to send out a form on Twitter/Discord/Reddit for potential speakers to sign up, I'd love some feedback on the form :100: 

Form link: https://forms.gle/BS4o3LGnkJg5kHG26

cc @kvark @cwfitzgerald @alice-i-cecile @erlend-sh

**Comment by AngelOnFira** [comment]
The form is live, and I've posted to Reddit, Twitter, and several Discord servers :)

**Comment by lukors** [comment]
Didn't see this discussion until now, so maybe kind of late for feedback on the form. Either way, some things pop up in my mind when reading this form. I hope they are helpful! :)

1. The form does not clarify what will happen after you submit it. For instance, should you expect an email letting you know if you have been accepted?
2. It might be good is to provide space for a short description of the talk and not just a title. If I were submitting, I would want to give more context than just a title.
3. It could use a more precise explanation of the type of talks you're interested in. There is an explanation here in this discussion, but it would be good to have it on the form, as it's not sure people will have read this discussion.
4. Is the submitter going to present the talk live, pre-recorded, or either?
5. Assuming it's a live broadcast, will there be time for questions after the allotted time (how much?), or should you provide time for that yourself within the allotted time?
6. How strict are the times? Will you get your 15 minutes and nothing more? Is it inconvenient if the talk ends up shorter than the allotted time? It might be worth indicating your stance here.

I hope I'm not coming across as critical! Everything that is there already is good, this is a great initiative, and I'm looking forward to it! :D

**Reply by AngelOnFira** [reply]
This is a good point, I'm quite notoriously bad at getting back to people who sign up for the gamedev meetup :sweat_smile: 

Just to write down a few thoughts:

- re: streaming; preferably live, it will be streamed from StreamYard. I think I could allow for pre-recorded, but I really like the dynamics of the existing live Rust Gamedev meetup. I feel like pre-recorded might make it feel more like a conference/bigger thing than I'm aiming for. Maybe in the future it would be good to try and plan something larger for anything within the Rust gamedev ecosystem?
- re: timeslots; I was aiming to use the time allocations as "guidelines", then be a bit more open with how long questions can run for. But that doesn't really work well for putting out a schedule of when speakers are expected to present. I'll likely reach out to speakers and figure out how long they need. I think my main intention here was to open the doors to people who feel like they want to say something, but don't want to feel the pressure of trying to talk for 15/30 minutes (hence the "lightning" talks).

I think it would be good for me to make a page somewhere on gamedev.rs that describes the intentions more. I really like low-overhead, relaxed meetups, but just because I understand how it's low effort, doesn't mean it's like this for the presenters.

Anyways, I'll update the form with some of your suggestions, and just reach out to the currently-submitted speakers with the extra questions :100:

**Comment by lukors** [comment]
I think it should mention the location of the event too. :)

**Reply by AngelOnFira** [reply]
Thanks for this ping, I completely forgot to send out emails :sweat_smile: you are definitely right about location as well!
